modules/alloy/src/main/ts/ephox/alloy/positioning/layout/LayoutInset.ts

```diff
@@ -15,7 +15,7 @@ import { Placement, getPlacement } from './Placement';
   `LayoutInset.north` will appear horizontally centered overlapping the top of the anchor.
  */
 
-const labelPrefix = 'layout-inner';
+const labelPrefix = 'layout-inset';
```

Comment:
I missed this in the previous rename PR, so just fixing now (it's only used for debugging at this stage anyways).